appendices/VK_EXT_image_drm_format_modifier.txt

```diff
@@ -449,8 +449,7 @@ _requires_ the application to set pname:size to 0.
   . [[VK_EXT_image_drm_format_modifier-fn3]]
     https://www.khronos.org/registry/EGL/extensions/MESA/EGL_MESA_image_dma_buf_export.txt[`EGL_MESA_image_dma_buf_export`]
   . [[VK_EXT_image_drm_format_modifier-fn4]]
-    https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/include/uapi/drm/drm_mode.h?id=refs/tags/v4.10#n392[`struct
-    drm_mode_fb_cmd2`]
+    link:++https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/include/uapi/drm/drm_mode.h?id=refs/tags/v4.10#n392++[`struct drm_mode_fb_cmd2`]
```

Comment:
What's the purpose of this change?

Reply:
I guess nothing. Link checker I used didn't like something about that link, but this didn't help.